dashboard/client/src/App.tsx

```diff
@@ -169,8 +171,25 @@ const App = () => {
                   <Route element={<Navigate replace to="overview" />} path="" />
                   <Route element={<OverviewPage />} path="overview" />
                   <Route element={<NewIAClusterPage />} path="cluster">
-                    <Route element={<Nodes newIA />} path="" />
-                    <Route element={<NodeDetailPage />} path="nodes/:id" />
+                    <Route element={<ClusterLayout />} path="">
+                      <Route
+                        element={
+                          <SideTabPage tabId="info">
+                            <ClusterDetailInfoPage />
+                          </SideTabPage>
+                        }
+                        path="info"
+                      />
+                      <Route
+                        element={
+                          <SideTabPage tabId="charts">
+                            <Nodes newIA />
+                          </SideTabPage>
+                        }
+                        path=""
+                      />
+                      <Route element={<NodeDetailPage />} path="nodes/:id" />
```

Comment:
Does the side tab for the clusters layout show up on this page? Seems like I need to create a way to hide the side tab for certain child routes.

Reply:
yes. it still shows up.

Reply:
I think it is better this is shown up? So I can easily jump back and fourth?

Reply:
That doesn't work because sometimes both the parent page and a child page has a side tab.

For example Clusters layout has a sidebar and Clusters > Node Detail Layout will have a sidebar. In that case we will show both side bars at the same time which will be very ugly.

I think you can merge this PR first and I can take a stab at implementing the multiple side bar behavior correctly.

Reply:
sgtm! I just merged it, so feel free to follow up!